tests/functional/pages/introduction.page.js

```diff
@@ -6,6 +6,10 @@ class IntroductionPage extends BasePage {
     super('introduction-page');
   }
 
+  myAccountLink() {
```

Comment:
you could push this to `BasePage` so it doesn't need to be duplicated?

Reply:
It's not there on the error page.  Is it okay to move it there if it's not on literally every page?